test/README

```diff
@@ -15,3 +15,7 @@ $ make
 $ cd test/<platform_name>
 $ make check-valgrind
 
+To run these tests with gdb, use the following libtool command:
+        libtool --mode=execute gdb ./<test_name>
```

Comment:
To be more consistent with the valgrind instructions above, please format this line:
`$ libtool --mode=execute gdb ./<test_name>`
and add empty line after this.

Otherwise looking good. You can add my reviewed-by tag to the commit log.